.circleci/config.yml

```diff
@@ -32,3 +32,26 @@ jobs:
         root: .
         paths:
           - ./e2ectl 
+
+  github-release:
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/e2ectl
+    machine: true
+    steps:
+      - checkout
+
+      - attach_workspace:
+          at: .
+
+      - run: ./github-release.sh $CIRCLE_PROJECT_REPONAME $CIRCLE_SHA1 $PERSONAL_ACCESS_TOKEN
```

Comment:
@TheoBrigitte WDYT about extracting architect command to do a release? We are doing it already for tagged commits.

Reply:
I chose consistency with other GS Go projects.

Curious was it considered to use/switch to Go community defacto standard - https://goreleaser.com/ ?

Reply:
> Curious was it considered to use/switch to Go community defacto standard - https://goreleaser.com/ ?

I don't think so. But I'd like to see us using it. Especially if this is encapsulated into an orb.